File: samples/go/08objects/objects.go
package main

import (
	"main/circle"
	"main/rectangle"
	"main/square"
	"main/trapezoid"
	"main/triangle"
)

func main() {
	r := rectangle.New(5, 2)
	rectangle.Print(r)
	s := square.New(5)
	square.Print(s)
	c := circle.New(3)
	circle.Print(c)
	t := triangle.New(5, 2)
	triangle.Print(t)
	p := trapezoid.New(5, 2, 3)
	trapezoid.Print(p)
}

File: samples/kotlin/08objects/objects.kt
import shapes.Rectangle;
import shapes.Square;
import shapes.Triangle;
import shapes.Circle;
import shapes.Trapezoid;

fun main() {
    val rectangle = Rectangle(5.0, 2.0);
    val square = Square(5.0);
    val triangle = Triangle(4.0, 2.0);
    val circle = Circle(3.0);
    val trapezoid = Trapezoid(3.0, 5.0, 2.0);

    println(rectangle);
    println(square);
    println(triangle);
    println(circle);
    println(trapezoid);
}